src/openklant/tests/test_migrate.py

```diff
@@ -689,3 +693,74 @@ def test_rsin(self):
         )
 
         self.assertEqual(output, [f"{self.live_server_url}{partij_url}"])
+
+    def test_klant_with_telefoonnummer_creates_digitaaladres(self):
+        # Klant with telefoonnummer creates a new DigitaalAdres
+        klant_with_phone = Klant(telefoonnummer="0612345678")
+
+        subject = Partij(nummer="12345", soort_partij=SoortPartij.persoon)
```

Comment:
Instead of instantiating these directly, you'll probably want to use factories. I don't think you'll have to instantiate the klant, because the klant data is retrieved with a request